packages/modeling/src/operations/extrusions/slice/equals.js

```diff
@@ -4,7 +4,7 @@ const vec3 = require('../../../maths/vec3')
  * Determine if the given slices are equal.
```

Comment:
Maybe a better definition of what "equal" means here.

Reply:
Done